Prose/ProseLib/Sources/ConversationFeature/Atoms/DaySeparator.swift

```diff
@@ -20,12 +21,12 @@ struct DaySeparator: View {
 
     var body: some View {
         HStack {
-            Color.borderTertiary
+            Asset.Color.Border.tertiary.swiftUIColor
                 .frame(height: 1)
             Text(date, formatter: Self.formatter)
                 .layoutPriority(1)
                 .foregroundColor(.secondary)
-            Color.borderTertiary
+            Asset.Color.Border.tertiary.swiftUIColor
```

Comment:
I don't like this syntax… I just have no choice because swiftgen generates colors like this 😕 I'd prefer `Colors.Border.tertiary` being a SwiftUI `Color`.

I could create a generation template for it, but I don't think it's an issue worth giving this much time right now.

Reply:
It's not that hard. Here's an example: https://gist.github.com/nesium/5848a6a95653f98a8236f5ecc707ee6b

Reply:
Do you want me to do this in this PR?

Reply:
It would be nice I guess. We could live with it but it feels like a regression. I was just thinking how we'd get to the UI- or NSColor if we'd keep the current syntax (extension on SwiftUI.Color). In that case we'd probably need to generate all three of them (extension on Color, UIColor and NSColor). I doesn't sound that bad to me since the number of colors (and the growth in generated code) will be limited.

Reply:
> It would be nice I guess.

Ok, so I'll do it tonight and finish my work on stores tomorrow morning I guess.

When I said

> I'd prefer `Colors.Border.tertiary` being a SwiftUI Color.

I meant "The most readable would be …", but in fact the most practical would be `Colors.Border.tertiary.(color|uiColor|nsColor)`. What do you think? Is it short enough compared to `(Color)?.borderTertiary`?

Or do you want extensions on Color too?

Reply:
That's true. I was just aiming for the syntax where you can skip the first `Color` part when passing it as an argument.

It's really syntactic sugar anyway and at this point I'd say either leave it as-is (maybe drop the `swiftUI` from `swiftUIColor`) or make it nice all the way. You have written all the SwiftUI code so far, you probably have a better idea how much this changes affects readability of the code.

Reply:
For the `swiftUI` prefix, I had no choice, the current generator created `color` 😕

I think having `Colors.Border.tertiary.color` is fine, it's like `L10n` but with a `.color` suffix.

Also, I think it makes the distinction between SwiftUI colors and our colors explicit, and avoid name conflicts. We can always do `fileprivate colors = Color.Some.Very.Long.Prefix.self` to shorten the calls in a file, as I did with `L10n`.

WDYT?

Reply:
Ok!